src-self-hosted/main.zig

```diff
@@ -682,13 +682,13 @@ pub fn cmdFmt(gpa: *Allocator, args: []const []const u8) !void {
             process.exit(1);
         }
         if (check_flag) {
-            const anything_changed = try std.zig.render(gpa, io.null_out_stream, tree);
+            const anything_changed = try std.zig.render(gpa, &io.null_out_stream, tree);
```

Comment:
The `&` on these can be reverted now right? Other than that this looks ready to merge now.